src/Microsoft.FeatureManagement/FeatureManagementOptions.cs

```diff
@@ -11,6 +11,7 @@ public class FeatureManagementOptions
         /// <summary>
         /// Controls the behavior of feature evaluation when dependent feature filters are missing.
         /// If missing feature filters are not ignored an exception will be thrown when attempting to evaluate a feature that depends on a missing feature filter.
+        /// This option will throw an error if used in combination with RequirementType.All as the execution may unintentionally enable a flag.
```

Comment:
"This option will throw an error" It's not that the option throws. The option is invalid when combined with `all`.

Reply:
You should be able to use `cref` I believe to link to the `RequirementType.All` type.